release-team/role-handbooks/enhancements/README.md

```diff
@@ -236,15 +284,15 @@ This enhancement is targeting for stage `<insert-stage-here>` for {current relea
 Here's where this enhancement currently stands:
 
 - [ ] KEP readme using the [latest template](https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template) has been merged into the k/enhancements repo.
-- [ ] KEP status is marked as `implementable` for `latest-milestone: { CURRENT_RELEASE }`
-- [ ] KEP readme has a [updated detailed test plan section](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md?plain=1#L257-L328) filled out
-- [ ] KEP readme has up to date graduation criteria
-- [ ] KEP has a production readiness review that has been completed and merged into k/enhancements.
+- [ ] KEP status is marked as `implementable` for `latest-milestone: { CURRENT_RELEASE }`. KEPs targeting `stable` will need to be marked as `implemented` after code PRs are merged and the feature gates are removed.
+- [ ] KEP readme has up-to-date graduation criteria
+- [ ] KEP has [submitted a production readiness review](https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md#submitting-a-kep-for-production-readiness-approval) request for approval and has a reviewer assigned.
+- [ ] KEP has a production readiness review that has been completed and merged into k/enhancements. 
```

Comment:
@npolshakova, same as what I suggested here - https://github.com/kubernetes/enhancements/pull/4162#issuecomment-1695689592

What do you think? :slightly_smiling_face:

Reply:
Yep, we can combine this new one with the existing `KEP has a production readiness review that has been completed and merged into k/enhancements.` and link to the new documentation!